### ISSUE TITLE
Improve table recognition (renamed)

### ISSUE THREAD

**Issue by rgosens2** [fmalina/PDFtranscript#15]
It cannot create html tables from tables in the pdf and totally skips 'em, including the text. Not very useful.

Tested on Intel manual (see: Attached file)

[vol2a_2016_test1.pdf](https://github.com/fmalina/unilex-transcript/files/7775001/vol2a_2016_test1.pdf)

**Comment by fmalina** [comment]
Interesting `TABLES = 1` is turned on by default, did you turn it off?

**Comment by fmalina** [comment]
I confirm this is a bug and added your document to default test suite

**Comment by fmalina** [comment]
I added your document to the default test suite:

Before https://fmalina.github.io/unilex-transcript/tests/PDF/vol2a.pdf
After https://fmalina.github.io/unilex-transcript/tests/HTM/vol2a.htm

I see the tables are there including text, just search for "ASCII adjust AL after addition", which is the last cell in the first table. Table formatting was dropped for this table. I suppose `grid_data` for `reconstruct_table` didn't catch the grid.

**Comment by rgosens2** [comment]
Thanks for looking into it. I see the Agincourt House pdf gives proper tables, so it can be done. Intel's pdf must be a bit trickier.

**Comment by airtonix** [comment]
@fmalina are we saying that it does output `<table/>` or not ? 

because i loaded up your example output there and it seems that tables end up like this: 

![image](https://github.com/fmalina/unilex-transcript/assets/61225/b8b22f2c-0cec-4ee4-b41a-70f8929ec09b)

**Comment by fmalina** [comment]
Yes, tables are supported and work in some examples. Obviously yours didn’t get recognised and there are various resons why it happens

**Comment by airtonix** [comment]
> Yes, tables are supported and work in some examples. Obviously yours didn’t get recognised and there are various resons why it happens

Ok so i'm not saying there's a problem with my documents (there very well might be though).

What I am saying is: 

your example is broken.

That screenshot is from your HTML output: 

https://github.com/fmalina/unilex-transcript/blob/main/tests/HTM/vol2a.htm

there's no table tags in that html file. 

I ask because your comment: https://github.com/fmalina/unilex-transcript/issues/15#issuecomment-1007031705

seems to indicate that you're not aware that something is different now.
